docs/releasing.md

```diff
@@ -1,116 +1,141 @@
+  no value to the release, create a summary line in `Superseded` section with
+  the PR ids and summary title. This way thye are acknowledged to be part of
+  the release, but not overwhelming the actual changes contained by the release.
+- Save the draft.
 
-   To use your GPG signature when pushing the tag, use `git tag -s [...]` instead
-1. Push the tags to the GitHub repository:
+### Tagging the Ironic and Mariadb images
 
-   ```bash
-   git push origin v0.x.y
-   git push origin test/v0.x.y
-   ```
+NOTE: This part will be moved to BMO releasing after BMO release process and
+maintenance plan has been agreed on.
 
-   NB: `origin` should be the name of the remote pointing to
-   `github.com/metal3-io/cluster-api-provider-metal3`
-1. Run `make release` to build artifacts (the image is automatically built by CI)
-1. [Create a release in GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
-   that contains the elements listed above that have been created in the `out`
-   folder
-1. Create a branch `release-0.x` for a minor release for backports and bug fixes.
-   This is not needed for patch releases.
+Before publishing CAPM3 release, we need to tag corresponding
+[Ironic-image](https://github.com/metal3-io/ironic-image) and
+[Maridb-image](https://github.com/metal3-io/mariadb-image) repositories.
+Those repositories do not have separate releasing process, but for the sake of
+backwards compatibility, each of the CAPM3 releases is tagged in these
+repositories as well.
 
-### Permissions
+```bash
+# Ironic
+git clone git@github.com:metal3-io/ironic-image.git
+cd ironic-image
+# if retagging old tag, add capm3-v1.x.y^{} to the line, e.g.
+# git tag -s -a capm3-v1.x.y -m capm3-v1.x.y capm3-v1.x.y^{}
+# this reuses the tag and resolves it into a sha for tagging
+git tag -s -a capm3-v1.x.y -m capm3-v1.x.y
+git push origin capm3-v1.x.y
+
+# Mariadb
+git clone git@github.com:metal3-io/mariadb-image.git
+cd mariadb-image
+# if retagging old tag, add capm3-v1.x.y^{} to the line, e.g.
+# git tag -s -a capm3-v1.x.y -m capm3-v1.x.y capm3-v1.x.y^{}
+# this reuses the tag and resolves it into a sha for tagging
+git tag -s -a capm3-v1.x.y -m capm3-v1.x.y
+git push origin capm3-v1.x.y
+```
+
+After this, [Ironic](https://quay.io/repository/metal3-io/ironic) and
+[Mariadb](https://quay.io/repository/metal3-io/mariadb) images will be built
+automatically with `capm3-v1.x.y` tag in Quay.
+
+### Publishing
 
-Releasing requires a particular set of permissions.
+Go to the releases and edit the draft release created earlier:
 
-- Tag push access to the GitHub repository
-- GitHub Release creation access
+- Make sure Ironic and Mariadb version references are in place and of correct
+  version number.
+- If the release you're making isn't the actually latest release (new
+  minor/major or new patch release in the latest minor), uncheck the
```

Comment:
I think you meant: 

```suggestion
- If the release you're making isn't the actually latest release (new patch release in the latest-1 minor release series), uncheck the
```

Reply:
I don't get the point about `latest-1` ? 

Fixed the wording otherwise.

Reply:
The confusing part to me here is: if you release any major/minor release, that would always be the latest release and you do not need to uncheck the "latest release" button. 
But, if it is the patch release from the stable branch which is N-1 where N is latest stable branch (i.e you are publishing 1.3.5 patch release from release-1.3 but latest stable branch you have is release-1.4) then you need to uncheck it

Reply:
There is only one latest release per repo. In GitHub it means the release that is highlighted in the repository main page. 
The only time you tick the latest box is when you release from top off the latest release branch, otherwise must be unticked. Release from 1.2 branch will never be latest if you have 1.3 releases.

I'll see if there is better wording to make this clearer.